src/main/scala/is/hail/utils/ArrayBuilder.scala

```diff
@@ -36,6 +41,15 @@ class ArrayBuilder[@specialized T](initialCapacity: Int)(implicit tct: ClassTag[
     size_ += 1
   }
 
+  def ++=(a: Array[T]): Unit = ++=(a, a.length)
```

Comment:
remove `: Unit`

Reply:
It won't compile without the return type due to overloaded methods.